src/widget/wtracktableview.cpp

```diff
@@ -1994,3 +2015,18 @@ void WTrackTableView::keyNotationChanged()
     QWidget::update();
 }
 
+void WTrackTableView::sortColumnChanged(double v)
+{
+    int sortColumn = (int)v;
+    Qt::SortOrder sortOrder = m_pSortColumn->get() ? Qt::DescendingOrder : Qt::AscendingOrder;
+    horizontalHeader()->setSortIndicator(sortColumn, sortOrder);
```

Comment:
Both slots share the last two lines. Maybe extracting them as a separate function `afterSortingChanged(int, QtSortOrder)` could also help to avoid the duplicate sorting in `slotSortingChanged(int, QtSortOrder)`?

Minor note: I would prefer the naming convention `onSomethingHappened` for slots to distinguish them from signals named `somethingHappened`. But the naming was already inconsistent before in this class and we should keep just the 2 existing variants instead of introducing a 3rd one ;)